src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -104,6 +104,7 @@ struct Normalized : db::ComputeTag {
     }
     return vector;
   }
+  using type = typename Tag::type;
```

Comment:
This is not part of the interface, please don't add it.

Reply:
Let’s chat to figure this out...it’s there because my GH executable doesn’t compile without it

Reply:
To clarify: I added "using type = " here and below because I need to put `Tags::Normalized<Tags::UnnormalizedFaceNormal<>>` into a `Variables`, because I need the unit normal vector to be packaged with `package_data`. When I try to do this without these changes, I get compiler errors like the following:

~~~
In file included from /Users/geoffrey/Codes/spectre/spectre/src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp:14:
/Users/geoffrey/Codes/spectre/spectre/src/DataStructures/Variables.hpp:111:44: error: no member named 'type' in 'Tags::Normalized<Tags::UnnormalizedFaceNormal<1, Frame::Inertial> >'
           cpp17::is_same_v<typename Tags::type::type,
                                     ~~~~~~^
/Users/geoffrey/Codes/spectre/spectre/src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp:356:7: note: in instantiation of template class 'Variables<brigand::list<gr::Tags::SpacetimeMetric<1, Frame::Inertial, DataVector>, GeneralizedHarmonic::Tags::Pi<1, Frame::Inertial>, GeneralizedHarmonic::Tags::Phi<1, Frame::Inertial>, gr::Tags::Lapse<DataVector>, gr::Tags::Shift<1, Frame::Inertial, DataVector>, gr::Tags::InverseSpatialMetric<1, Frame::Inertial, DataVector>, GeneralizedHarmonic::Tags::ConstraintGamma1, GeneralizedHarmonic::Tags::ConstraintGamma2, Tags::Normalized<Tags::UnnormalizedFaceNormal<1, Frame::Inertial> > > >' requested here
      *packaged_data) = spacetime_metric;
      ^
/Users/geoffrey/Codes/spectre/spectre/src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp:519:38: note: in instantiation of member function 'GeneralizedHarmonic::UpwindFlux<1>::package_data' requested here
template struct GeneralizedHarmonic::UpwindFlux<1>;
~~~

What is the proper way for me to resolve this compiler error, @nilsdeppe, if not what I've done here, so that I can put `Tags::Normalized<Tags::UnnormalizedFaceNormal<>>` into a `Variables`?

Reply:
I'd like to consult in person with @wthrowe about what the best solution is but this is behaving exactly as it should. You are trying to add a compute tag to a Variables which doesn't make sense since a Variables stores data, not lazily evaluated functions. What we probably want is to turn `Tags::Normalized` into a simple tag and add `Tags::NormalizedCompute` as a compute tag, doing so in such a way that we can use the base tag mechanism to retrieve `Tags::NormalizedCompute` using `Tags::Normalized`. Does that make sense?

Reply:
I just realized this exact same issue is the subject of #1496